client/login/wp-login/index.js

```diff
@@ -93,22 +110,13 @@ class Login extends React.Component {
 			<Main className="wp-login">
 				{ this.magicLoginMainContent() || (
 					<div>
-						<div className="wp-login__header">
-							<Gridicon icon="user-circle" size={ 72 } />
-							<div>{
-								// @TODO show currently logged in user if any
-								translate( 'You are signed out' )
-							}</div>
-						</div>
 						<div className="wp-login__container">
 							{ magicLoginView === REQUEST_FORM
 								? <RequestLoginEmailForm />
-								: <LoginBlock title={ translate( 'Sign in to WordPress.com' ) } />
+								: <LoginBlock title={ translate( 'Log in to your account.' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Log in to your WordPress.com account' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).